dvc/fs/webhdfs.py

```diff
@@ -66,115 +32,71 @@ def _get_kwargs_from_urls(urlpath):
             )
         )
 
-    @wrap_prop(threading.Lock())
-    @cached_property
-    def hdfs_client(self):
-        import hdfs
+    def _parse_config(self, path=None, alias=None):
+        import configparser
 
-        logger.debug("HDFSConfig: %s", self.hdfscli_config)
+        from hdfs.config import Config
+        from hdfs.util import HdfsError
 
         try:
-            return hdfs.config.Config(self.hdfscli_config).get_client(
-                self.alias
-            )
-        except hdfs.util.HdfsError as exc:
-            exc_msg = str(exc)
-            errors = (
-                "No alias specified",
-                "Invalid configuration file",
-                f"Alias {self.alias} not found",
-            )
-            if not any(err in exc_msg for err in errors):
-                raise
-
-            http_url = f"http://{self.host}:{self.port}"
-            logger.debug("URL: %s", http_url)
-
-            if self.token is not None:
-                client = hdfs.TokenClient(http_url, token=self.token, root="/")
-            else:
-                client = hdfs.InsecureClient(
-                    http_url, user=self.user, root="/"
+            config = Config(path)
+        except HdfsError:
+            return None
```

Comment:
For the record: discussed that we might just drop hdfscli config, since we are no longer using it as a backend.